pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -709,6 +709,26 @@ default void delete(String topic, boolean force) throws PulsarAdminException {
      */
     CompletableFuture<MessageId> terminateTopicAsync(String topic);
 
+    /**
+     * Terminate the partitioned topic and prevent any more messages being published on it.
+     * <p/>
+     *
+     * @param topic
+     *            topic name
+     * @return the message id of the last message that was published in the each partition of topic
+     */
+    List<MessageId> terminatePartitionedTopic(String topic) throws PulsarAdminException;
```

Comment:
Is it better to return a Map?
Because it is hard to understand the contents of the list, especially in case of null values